### ISSUE TITLE
Interested people option besides RSVP for event

### ISSUE THREAD

**Issue by CleverFool77** [publiclab/plots2#5177]
## Description
Whenever the person creates a event. There is only one option given which RSVP for attending event. 
Rather there should be another feature for those people who are interested but unsure that whether they'll attend event or not.

**Comment by CleverFool77** [comment]
@publiclab/plots2-reviewers Is this issue correct ?
If yes, then I would like to work on this.
Thanks. :)

**Comment by jywarren** [comment]
Cool idea! Shall we start with a visual mockup of what it would look like?
Thank you!!!

**Comment by CleverFool77** [comment]
How about giving an option beside RSVP ?

**Comment by IshaGupta18** [comment]
Yes I think that's what @jywarren means here. I would like to help you out in this one @CleverFool77 . We could divide this up, I could do the backend of it and you could do the UI, or vice-versa, whatever suits best! What say?

**Comment by CleverFool77** [comment]
Hi @IshaGupta18 I would really love your help but I would like to try a backend code in this project for once. Till now I've been doing frontend part and I had wanted to explore some other features too. I know that It would be slow but I wanted to try it out. I'll let you know when I require your help. For now I would like to try this myself. :smile:

**Comment by IshaGupta18** [comment]
Absolutely, I think it will be good exposure, if you need any guidance with that, do let me know! So can I start with the front-end then?

**Comment by IshaGupta18** [comment]
@CleverFool77 please let me know if you're okay with it?

**Comment by CleverFool77** [comment]
Hi @IshaGupta18, Sorry for the late reply, I went offline back yesterday. So I wanted to say that would you mind if I go for these parts together first. I waana give it a try first. And then I would ask you for help and guidance which I might be surely needing it. :sweat_smile:

**Comment by IshaGupta18** [comment]
Sure! Go right ahead @CleverFool77

**Comment by CleverFool77** [comment]
Thanks @IshaGupta18  Though I'm curious as you said you can proceed with frontend part. So how is the work is checked as the we need back-end part for doing changes for frontend. :thinking:

**Comment by IshaGupta18** [comment]
The flow will be: There will be a button on the front-end saying "Interested?", on clicking this button, a function in the respective controller should be called, where a variable is defined to keep a count of the number of interested people, and it will be incremented. This number will then be displayed alongside the button. JS can also be used here to do the same, but if we might want to do something with that variable later, a controller function will be more suitable. Does this make sense?

**Comment by IshaGupta18** [comment]
So I would have basically created that button and helped you out the the background flow, that's what I meant.

**Comment by CleverFool77** [comment]
Thank you so much for your explanation. @IshaGupta18 :wink: 
I would ask you out for help in future. :)
